### PR TITLE
RGBcolorgame.css

### PR DESCRIPTION
**Pull request by RS-labhub**

### CODE REVIEW

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
Please provide html file as well.

**Comment by RS-labhub** [comment]
> Please provide html file as well.

Providing

**Review by aayush105 — COMMENTED** [review]
@RS-labhub add the output of your code in image folder as well as in the README.md file and push it again.

**Comment by RS-labhub** [comment]
> @RS-labhub add the output of your code in image folder as well as in the README.md file and push it again.

Okayyyy...

File: RGBcolorgame.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
+body {
```
[the remark this answers is not in the document]
Comment (RS-labhub) [reply RGBcolorgame.css:1]: Done!

**Comment by RS-labhub** [comment]
> @RS-labhub add the output of your code in image folder as well as in the README.md file and push it again.

@aayush105 please take a look. Other files are uploaded from another patch unknowingly.

**Comment by aayush105** [comment]
> > @RS-labhub add the output of your code in image folder as well as in the README.md file and push it again.
> 
> @aayush105 please take a look. Other files are uploaded from another patch unknowingly.

Dont add in such way add all in same PR. This time i will merge but from next time it will not be merged.
Thank you!

**Comment by RS-labhub** [comment]
> > > @RS-labhub add the output of your code in image folder as well as in the README.md file and push it again.
> > 
> > 
> > @aayush105 please take a look. Other files are uploaded from another patch unknowingly.
> 
> Dont add in such way add all in same PR. This time i will merge but from next time it will not be merged. Thank you!

Thank you! I'll take care from the next time... Thank you so much!

**Pull request merged**